db/migrate/20160607141810_remove_database_synchronization_role.rb

```diff
@@ -0,0 +1,15 @@
+class RemoveDatabaseSynchronizationRole < ActiveRecord::Migration[5.0]
+  class ServerRole < ActiveRecord::Base; end
+  class MiqServer < ActiveRecord::Base; end
+  class AssignedServerRole < ActiveRecord::Base; end
+
+  def up
+    say_with_time("Removing database synchronization role") do
```

Comment:
You also have to remove the role from the settings.  Settings.server.role may contain database_synchronization.  Not sure if you want to do it here or in the other migration where you manipulate the settings table.